lib/win32/taskscheduler.rb

```diff
@@ -193,7 +193,30 @@ def enum
     # Returns whether or not the specified task exists.
     #
     def exists?(task)
```

Comment:
I'd probably change `task` here to something like `full_task_path` so you aren't using task for two different things in this method (the fully qualified task path at first, and then an object returned by `root.GetTask()` later.)